source/stylesheets/base/_feature_sections.scss

```diff
@@ -21,32 +21,44 @@
 
     .illustration {
       align-items: flex-end;
-      justify-content: center;
+      justify-content: flex-end;
     }
 
     .text-and-description {
       @include grid-column(6 of 12);
       align-items: center;
       display: flex;
     }
+
+    h1 {
+      font-size: 3.1em;
+    }
+
+    time {
+      margin-bottom: 10px;
+    }
   }
 }
 
 .featured-flavors {
-  background: url("../../images/linedpaper.png");
+  @include clearfix;
+  background-color: $white;
 
   .container {
-    height: 500px;
     padding: 50px;
```

Comment:
It might be worth establishing some spacing variables. That way your design will feel tighter and more consistent.

Reply:
I ended up taking out the heights because of the clearfix mixin you showed me, so just forgot about this one. This way the content will be the decider of the heights